### PR TITLE
enhance(main/cmus): enable modplug and wavpack input plugins

### PR DESCRIPTION
**Pull request by pgaskin**
The dependencies weren't around when the cmus package was added.

### CODE REVIEW

**Review by twaik — APPROVED** [review]
LGTM. @truboxl, Probably you have objection about the order of `TERMUX_PKG_DEPENDS` items?

File: packages/cmus/build.sh [hunk 1]
```diff
@@ -3,7 +3,8 @@ TERMUX_PKG_DESCRIPTION="Small, fast and powerful console music player"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="2.12.0"
-TERMUX_PKG_DEPENDS="libandroid-support, libiconv, ncurses, pulseaudio, ffmpeg, libmad, opusfile, libflac, libvorbis"
+TERMUX_PKG_REVISION="1"
+TERMUX_PKG_DEPENDS="libandroid-support, libiconv, ncurses, pulseaudio, ffmpeg, libmad, libmodplug, opusfile, libflac, libvorbis, wavpack"
```
Comment (truboxl) [inline packages/cmus/build.sh:7]:
```suggestion
TERMUX_PKG_DEPENDS="ffmpeg, libandroid-support, libflac, libiconv, libmad, libmodplug, libvorbis, libwavpack, ncurses, opusfile, pulseaudio"
```

File: packages/cmus/build.sh [hunk 2]
```diff
@@ -3,7 +3,8 @@ TERMUX_PKG_DESCRIPTION="Small, fast and powerful console music player"
...
 TERMUX_PKG_SRCURL=https://github.com/cmus/cmus/archive/v${TERMUX_PKG_VERSION}.tar.gz
```
Comment (truboxl) [inline packages/cmus/build.sh:8]:
```suggestion
TERMUX_PKG_SRCURL=https://github.com/cmus/cmus/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

**Comment by twaik** [comment]
I will merge it in 24 hours if no objections or after it gets 2+ approves.

**Comment by pgaskin** [comment]
bump

**Comment by twaik** [comment]
Oh, right.

**Pull request merged**